.github/workflows/build.yml

```diff
@@ -0,0 +1,113 @@
+name: Scala tests with coverage
+
+on:
+  pull_request:
+    paths-ignore: ['**.md']
+  push:
+    paths-ignore: ['**.md']
+
+jobs:
+  test:
+
+    runs-on: ubuntu-20.04
+
+    env:
+      SBT_OPTS: >-
+        -Xmx3g
+        -Denv.type=test
+        -Ddirectory.url=ldap://localhost:3389 -Ddirectory.password=testtesttest
+        -Dpostgres.host=localhost -Dpostgres.port=5432
```

Comment:
any reason for swapping to port 5432 from 15342?

Reply:
oh maybe this was why travis used 15432 instead of postgres's default 5432 port:
> Port 15432 is used here for the local machine, rather than using 5432, in case an instance of PostgreSQL was also running on the local machine. If an instance of PostgreSQL was running on the local machine and the same port was used, setting up the connection would fail as the port would already be in use.
https://www.openshift.com/blog/openshift-connecting-database-using-port-forwarding

this probably isn't an issue for us here though

Reply:
This ^. Seemed like not an issue for GHA